navigator-menu/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "navigator-menu",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-icons": "^4.4.0",
+    "react-router-dom": "^6.3.0",
+    "tailwindcss": "^3.1.6"
```

Comment:
`tailwindcss` is actually dev dependency. You don't need it in the production bundle.